plugins/checklist.md

```diff
@@ -16,17 +16,30 @@ The **Checklist** plugin helps the user keep track of all required actions by cr
 
 ## Try our Checklist demo
 
+In the {{site.productname}} editor, checklists are presented as lists with small checkboxes on the left hand side of the list items. After the item has been completed, a small tick or check mark is drawn in the box by clicking on it.
```

Comment:
Do checklists have their checkboxes on the left with RTL? Maybe "next to" is better?

Reply:
Yes, they do. I just moved this info from the bottom of the page, I didn't have time to read it :smile:.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1704/commits/447f952128e58b8de91df229610c884dbf529341